From d2cb3558dd14422a48a8966157d2dd27366fd61a Mon Sep 17 00:00:00 2001
From: Sujeet Baranwal <s-baranwal@ti.com>
Date: Mon, 14 Mar 2011 19:37:57 +0530
Subject: OMAP4: DSS: SW reset not required in dss init

In omap3 & omap4, soft reset is being done by HWMOD thus
not required to be done again in dss initialization
but reset is required in all the others.

This chnage is tested on omap4 and omap3 sdp and display
is working fine.

Change-Id: I725ab78b392e865047a2a385373f2ba88ff5832c
Signed-off-by: Sujeet Baranwal <s-baranwal@ti.com>
---
 arch/arm/mach-omap2/omap_hwmod_3xxx_data.c |    3 ++-
 drivers/video/omap2/dss/dss.c              |   11 -----------
 2 files changed, 2 insertions(+), 12 deletions(-)

Index: kernel/arch/arm/mach-omap2/omap_hwmod_3xxx_data.c
===================================================================
--- kernel.orig/arch/arm/mach-omap2/omap_hwmod_3xxx_data.c	2011-04-13 13:28:37.000000000 +0900
+++ kernel/arch/arm/mach-omap2/omap_hwmod_3xxx_data.c	2011-04-13 13:32:18.630855393 +0900
@@ -3179,7 +3179,8 @@
 	.rev_offs	= 0x0000,
 	.sysc_offs	= 0x0010,
 	.syss_offs	= 0x0014,
-	.sysc_flags	= (SYSC_HAS_SOFTRESET | SYSC_HAS_AUTOIDLE),
+	.sysc_flags	= (SYSC_HAS_SOFTRESET | SYSC_HAS_AUTOIDLE |
+				   SYSS_HAS_RESET_STATUS),
 	.sysc_fields	= &omap_hwmod_sysc_type1,
 };
 
